tools/influx_si/macros.xml

```diff
@@ -0,0 +1,265 @@
+                    <option value="i">instationary</option>
+                </param>
+                <when value="s"/>
+                <when value="i">
+                    <param type="data" name="input_ti" label="time course data" format="txt,tsv" multiple="true"/>
+                    <param name="time_order" type="select" label="--time_order" display="radio" optional="true" help="Time order for ODE solving (1 (default), 2 or 1,2). Order 2 is more precise but more time consuming. The value &#x27;1,2&#x27; makes to start solving the ODE with the first order scheme then continues with the order 2.">
+                        <option value="None">None</option>
+                        <option value="1">1</option>
+                        <option value="2">2</option>
+                        <option value="1,2">1,2</option>
+                    </param>
+                </when>
+            </conditional>
+            <section name="prlexp" title="Parallel labeling experiments" expanded="false">
+                <param optional="true" type="data" name="input_aux" format="ftbl" multiple="True" label="auxiliary FTBL(s): data only"/>
+            </section>
+            <section name="opt" title="Advanced Options" expanded="false">
+              <param argument="--noopt" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--noopt" optional="true" help="no optimization, just use free parameters as is (after a projection on feasibility domain), to calculate dependent fluxes, cumomers, stats and so on" />
+              <param argument="--noscale" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--noscale" optional="true" help="no scaling factors to optimize =&gt; all scaling factors are assumed to be 1" />
+              <param argument="--meth" type="select" label="--meth" optional="true" help="method for optimization, one of nlsic|BFGS|Nelder-Mead. Default: nlsic">
+                <option value="BFGS">BFGS</option>
+                <option value="Nelder-Mead">Nelder-Mead</option>
+                <option value="nlsic">nlsic</option>
+              </param>
+              <param argument="--fullsys" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--fullsys" optional="true" help="calculate all cumomer set (not just the reduced one necesary to simulate measurements)" />
+              <param argument="--emu" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--emu" optional="true" help="simulate labeling in EMU approach" />
+              <param argument="--irand" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--irand" optional="true" help="ignore initial approximation for free parameters (free fluxes and metabolite concentrations) from the FTBL file or from a dedicated file (cf --fseries and --iseries option) and use random values drawn uniformly from [0,1] interval" />
```

Comment:
the `label` should be a real text that describes that the parameter is doing, the `help` should extended help to the param.

If you use argument, the content of argument will be automatically attached, so you don't need to add `--irand` to the label or help.